glossary.md

```diff
@@ -1,5 +1,26 @@
 # Glossary
 
+## AMM
+
+Automated Market Maker, a smart contract facilitating token or yield exchange in
+an automated and trustless way between two parties.
+
+## Notional
+
+The total value of the futures contract when swapping yields. The notional of a
```

Comment:
The amount used to calculate settlement payments (e.g. if fixed - floating = 1% and notional = 1,000 USDC, then a payment of 1,000 USDC * 1% = 10 USDC is done). Before a user can trade the given notional size he first must pledge collateral in his margin account. The collateral amount is always smaller than the notional, giving the user leverage.

Reply:
I took your suggestion but dropped the equation part. Too much math may not be a good fit for the glossary.